### ISSUE TITLE
Remove need for app.json to specify pr-predestroy script

### ISSUE THREAD

**Issue by douglascayers** [heroku/salesforce-buildpack#35]
Just as [Heroku CI](https://devcenter.heroku.com/articles/heroku-ci-technical-detail-on-ci-lifecycle#step-3-applicable-buildpacks-are-invoked) will automatically look for `bin/test-compile` and `bin/test` scripts, if it's possible for Heroku during [review app destroy step](https://devcenter.heroku.com/articles/github-integration-review-apps#pr-predestroy-script) to look for something like `bin/pr-predestroy` script then that would allow the buildpack to provide a default and developers not to have this knowledge in their `app.json` unless they wanted to customize the behavior.

Currently, developers are adding the below snippet to their `app.json`, which relies on underlying knowledge of what artifacts this buildpack generates.

![image](https://user-images.githubusercontent.com/4142577/41565561-e61480d8-731c-11e8-95f7-504dfdd55a76.png)

**Comment by wadewegner** [comment]
I don't think this is possible, which is why I have it setup to use the `app.json`. I totally agree that it requires an absurd amount of knowledge and that it's easy to miss.

@ike-delorenzo and @devarispbrown, do you have an item on the roadmap to provide a better way we can hook this in, ideally through the buildpack itself so it doesn't require anything by the developer?
